src/apis/post/queries.ts

```diff
@@ -18,9 +18,16 @@ export const useGetInfinitePostsQuery = (params: GetPostsReq) =>
   useInfiniteQuery<GetPostsRes>({
     queryKey: ['getPosts'],
     queryFn: () => getPosts(params),
+    refetchOnMount: 'always',
     initialPageParam: null,
     getNextPageParam: lastPage =>
       lastPage?.hasNext
         ? lastPage.posts[lastPage.posts.length - 1].id
         : undefined
   })
+
+export const useGetSortOptions = (params: GetSortOptionsReq) =>
```

Comment:
```suggestion
export const useGetSortOptionsQuery = (params: GetSortOptionsReq) =>
```
postFix로 Query를 붙이고 있어서 붙이면 좋을 것 같습니다!

(SORT_OPTIONS 상수를 사용하면 해당 쿼리 사용되지 않을 수도 있을 것 같네요..!)

Reply:
해당쿼리 제거로 인해 스킵하겠습니다! ㅠㅠ